test/validate-pipeline.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+ROOT_DIR=$SCRIPT_DIR/..
+DEBUG_OUTPUT=${DEBUG_OUTPUT:-'/tmp/log.txt'}
+EVENT_TYPE=${EVENT_TYPE:-'push'}
+PIPELINE="${PIPELINE:-docker-build-rhtap}"
+TASK="${TASK:-update-deployment}"
```

Comment:
The same should apply for `deploy-check` task.
I think we should have a list of tasks to check.